bin/yaml2make

```diff
@@ -118,7 +126,7 @@ def read_file(test, type, run_index):
 
 def emit_make(test_spec, prefix):
     '''Emit a hash from the YAML test specification into a makefile that can be included'''
-    fh = tempfile.NamedTemporaryFile(mode='w', delete=False)
+    fh = tempfile.NamedTemporaryFile(mode='w', delete=False)        
```

Comment:
somewhat nitpicky here, but is there a way to automate removal of trailing whitespace when pushing with git? that always ends up with a nasty red highlight when git diff'ing

Reply:
I usually try to revert any files with just whitespace diffs, but missed this one.  I'll see if there is a general solution here.

Reply:
Fixed.